test/amplitude-client.js

```diff
@@ -4116,4 +4116,37 @@ describe('AmplitudeClient', function () {
       assert.equal(amplitude.options.apiEndpoint, constants.EVENT_LOG_EU_URL);
     });
   });
+
+  describe('custom library options', function () {
+    beforeEach(function () {
+      reset();
+    });
+    it('should use default library options', function () {
+      amplitude.init(apiKey);
+      amplitude.logEvent('Event Type 1');
+      const { name, version } = JSON.parse(queryString.parse(server.requests[0].requestBody).e)[0].library;
+      assert.equal(name, amplitude.options.library.name);
```

Comment:
nit: maybe we can also assert name as `amplitude-js` in this test, as this would be true in customized case as well.